backends/aptcc/apt-intf.cpp

```diff
@@ -174,6 +179,22 @@ bool AptIntf::init(gchar **localDebs)
     return m_cache->CheckDeps(AllowBroken);
 }
 
+bool AptIntf::isSystemDpkgConf() {
```

Comment:
That's a bit of a miseleading name - maybe something like `isDpkgForceConfSet` is a better name?